build.psm1

```diff
@@ -1410,6 +1410,11 @@ function Start-PSPackage {
         "AppImage" {
             if ($IsUbuntu14) {
                 Start-NativeExecution { bash -iex "$PSScriptRoot/tools/appimage.sh" }
+                $appImage = Get-Item PowerShell-*.AppImage
+                if ($appImage.Count -gt 1) {
+                    Write-Error "Found more than one AppImage package"
```

Comment:
Shall we stop in case this happens? If `$appImage` has more than one entries, then the `Rename-Item` below will fail.